### PR TITLE
Create docs build.sh

### PR DESCRIPTION
**Pull request by AyodeAwe**
This creates the required build script for the deployment of the docs generated in this repository.

### CODE REVIEW

File: ci/docs/build.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+for PROJECT in ${PROJECTS[@]}; do
+    if [ ! -d "api/$PROJECT/$BRANCH_VERSION" ]; then
+        mkdir -p api/$PROJECT/$BRANCH_VERSION
+    fi
+    rm -rf $DOCS_WORKSPACE/api/$PROJECT/$BRANCH_VERSION/*
+done
+
+
+mv $PROJECT_WORKSPACE/build/html/* $DOCS_WORKSPACE/api/deployment/$BRANCH_VERSION
```
Comment (AyodeAwe) [inline ci/docs/build.sh:29]: I'm not 100% sure the deployment docs should go in the [api section](https://docs.rapids.ai/api) of the website. I just followed with the same location of the other libraries.

Thoughts?

File: ci/docs/build.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+make html
+
+#Commit to Website
+cd $DOCS_WORKSPACE
+
+for PROJECT in ${PROJECTS[@]}; do
+    if [ ! -d "api/$PROJECT/$BRANCH_VERSION" ]; then
+        mkdir -p api/$PROJECT/$BRANCH_VERSION
+    fi
```
Comment (ajschmidt8) [inline ci/docs/build.sh:24]:
```suggestion
    mkdir -p api/$PROJECT/$BRANCH_VERSION
```

I know this was likely copy/pasted from another repo but we can simplify this logic here. `mkdir -p` will still exit with `0` if a folder already exists, so we don't need to guard it in an `if` statement.

File: ci/docs/build.sh (hunk 1 above)
Comment (ajschmidt8) [reply ci/docs/build.sh:29]: Based on Jacob's input, I think it should be fine to include this in the `/api` section of the site.

File: ci/docs/build.sh [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+# RAPIDS Deployment Docs build script for CI #
+#################################
+
+if [ -z "$PROJECT_WORKSPACE" ]; then
+    echo ">>>> ERROR: Could not detect PROJECT_WORKSPACE in environment"
+    echo ">>>> WARNING: This script contains git commands meant for automated building, do not run locally"
+    exit 1
+fi
+
+export DOCS_WORKSPACE="$WORKSPACE"
```
Comment (ajschmidt8) [inline ci/docs/build.sh:13]:
```suggestion
export DOCS_WORKSPACE="$WORKSPACE/docs"
```

this should be consistent with the `cudf` path below

- https://github.com/rapidsai/cudf/blob/288c81fd59d1b28541acb39235b314476f8a846a/ci/docs/build.sh#L13

**Review by ajschmidt8 — COMMENTED** [review]
Left a few minor comments. Also, I think we need to include the following statements to ensure that the `conda` environment is activated before we run `make html`. The `conda` environment will include all of the dependencies that are necessary to build the docs.

```sh
export PATH=/conda/bin:/usr/local/cuda/bin:$PATH
. /opt/conda/etc/profile.d/conda.sh
conda activate rapids
```

File: ci/docs/build.sh (hunk 3 above)
Comment (AyodeAwe) [reply ci/docs/build.sh:13]: I had initially had it that way but removed it as I realized that there is no `docs` directory in the root directory of this repository. Am I missing something?

Comment (ajschmidt8) [reply ci/docs/build.sh:13]: Yes, the Jenkins job below clones the https://github.com/rapidsai/docs/ repo into the `$WORKSPACE/docs` directory.

- https://6abc.com/northern-lights-forecast-philadelphia-pennsylvania-nj/12137915/

Comment (AyodeAwe) [reply ci/docs/build.sh:13]: Oops, thanks.

**Review by ajschmidt8 — APPROVED** [review]

**Pull request merged**